opencog/spacetime/octomap/atom_types_init.cc

```diff
@@ -0,0 +1,6 @@
+#include "opencog/spacetime/octomap/atom_types.definitions"
+
+#define INHERITANCE_FILE "opencog/spacetime/octomap/atom_types.inheritance"
+#define INITNAME octomap_types_init
+
+#include <opencog/atoms/proto/atom_types.cc>
```

Comment:
this is now `<opencog/atoms/atom_types/atom_types.cc>`

Reply:
Oh, I thought I was using the latest atomspace. Let me fix that.